cmd/kots/cli/install.go

```diff
@@ -162,6 +162,28 @@ func InstallCmd() *cobra.Command {
 			}()
 
 			upstream := pull.RewriteUpstream(args[0])
+			preferredChannelSlug, err := extractPreferredChannelSlug(upstream)
+			if err != nil {
+				return errors.Wrap(err, "failed to extract preferred channel slug")
+			}
+
+			// fetch the latest version of the license to verify channel access if online install
+			if license != nil {
+				if isAirgap && haveMultiChannelLicense(license) && !slugInLicenseChannels(preferredChannelSlug, license) {
+					return errors.New("requested channel not found in supplied license")
+				}
+				log.ActionWithSpinner("Checking for license update")
+				updatedLicense, err := replicatedapp.GetLatestLicense(license)
```

Comment:
wouldn't this try to get the latest license from replicated app even in airgap mode?

Reply:
yep, totally right